### PR TITLE
added nonstatic objects

### PR DESCRIPTION
**Pull request by Senka2112**

### CODE REVIEW

File: robotino_bringup/config/logitech.launch [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<launch>
```
Comment (NadiaHG) [inline robotino_bringup/config/logitech.launch]: could you please  move the launch file to the `robotino_bringup/launch` folder?

**Comment by ipa-nhg** [comment]
Hi @Senka2112 , what is the purpose of the camera? should it be included as a part of the robotino model? I mean, add the head_camera frame to the tf tree of the robot?

**Comment by Senka2112** [comment]
Hi Nadia
so  this is from my robotino (the borrowed one)
I ll remove it. Does not have to be in this  repo at all

Senka

On Wed, Mar 9, 2016 at 8:45 AM, Nadia Hammoudeh García <
notifications@github.com> wrote:

> Hi @Senka2112 https://github.com/Senka2112 , what is the purpose of the
> camera? should it be included as a part of the robotino model? I mean, add
> the head_camera frame to the tf tree of the robot?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/squirrel-project/squirrel_robotino/pull/97#issuecomment-194163467
> .

## 

Senka Krivić

mail:     krivic.senka@gmail.com
skype:  krivic.senka
mobile:  +38761278509
            +436606303207

**Comment by Senka2112** [comment]
done

On Wed, Mar 9, 2016 at 1:31 PM, Senka Krivić krivic.senka@gmail.com wrote:

> Hi Nadia
> so  this is from my robotino (the borrowed one)
> I ll remove it. Does not have to be in this  repo at all
> 
> Senka
> 
> On Wed, Mar 9, 2016 at 8:45 AM, Nadia Hammoudeh García <
> notifications@github.com> wrote:
> 
> > Hi @Senka2112 https://github.com/Senka2112 , what is the purpose of
> > the camera? should it be included as a part of the robotino model? I mean,
> > add the head_camera frame to the tf tree of the robot?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/squirrel-project/squirrel_robotino/pull/97#issuecomment-194163467
> > .
> 
> ## 
> 
> Senka Krivić
> 
> mail:     krivic.senka@gmail.com
> skype:  krivic.senka
> mobile:  +38761278509
>             +436606303207

## 

Senka Krivić

mail:     krivic.senka@gmail.com
skype:  krivic.senka
mobile:  +38761278509
            +436606303207

**Comment by ipa-nhg** [comment]
Thanks @Senka2112 

LGTM :+1:

**Pull request merged**